dll/win32/iernonce/iernonce.cpp

```diff
@@ -45,11 +45,15 @@ RunOnceExProcess(_In_ HWND hwnd,
 
     if (RunonceExInst_LM.m_bSuccess)
     {
-        // TODO: continue coding here
+        ProgressDlg dlg(RunonceExInst_LM);
+
+        dlg.RunDialogBox();
     }
 
     if (RunonceExInst_CU.m_bSuccess)
     {
-        // TODO: continue coding here
+        ProgressDlg dlg(RunonceExInst_CU);
+
+        dlg.RunDialogBox();
```

Comment:
So, on Windows, the runonce entries for HKLM and for HKCU, also use separate dialog boxes?

Reply:
Yes,🤣